theme-settings.php

```diff
@@ -150,6 +136,64 @@ function megatron_form_system_theme_settings_alter(&$form, &$form_state) {
 );
 */
 
+  /** CLF NAVIGATION OPTIONS
+  ---------------------------------------------------------- */
+  $form['clf_navigation_option'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Navigation Option'),
+    '#prefix' => '<div class="clf_navigation_option">',
+    '#suffix' => '</div>',
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#weight' => -7,
+  );
+
+  $form['clf_navigation_option']['clf_navigation_placement'] = array(
+    '#type' => 'select',
+    '#title' => t('Choose the type of primary navigation that should be used on this website.'),
```

Comment:
This may be better to have as a separate option related to drawer only because the primary can be left there and have a drawer with the region there now. hmmm, thoughts?